identity-service/src/captchaMiddleware.js

```diff
@@ -0,0 +1,30 @@
+const models = require('./models')
+
+async function captchaMiddleware (req, res, next) {
+  const libs = req.app.get('audiusLibs')
+
+  let score, ok, hostname
+  if (req.body.token) {
+    ({ score, ok, hostname } = await libs.captcha.verify(req.body.token))
```

Comment:
middlewares is a nice place to put this, but we don't want to block for now -- can we still fire this off async? (just not await it)

Reply:
yes!